presto-main/src/main/java/com/facebook/presto/operator/ExchangeClient.java

```diff
@@ -103,7 +106,7 @@
     @GuardedBy("this")
     private long successfulRequests;
     @GuardedBy("this")
-    private long averageBytesPerRequest;
+    private final ExponentialMovingAverage averageBytesPerRequestEMA;
```

Comment:
How about simply calling it `averageBytesPerRequest`? We usually trying to avoid abbreviations or acronyms in names.

Reply:
We use averageBytesPerRequest on the stack when we actually retrieve the value. The EMA is not the value it's the EMA.

I don't really have any preference for how to resolve this collision. I can stop abbreviating or call the thing on the stack something else (but what?)

Reply:
Renamed it, see if you like it.